recipes/libtheora/meta.yaml

```diff
@@ -44,6 +44,7 @@ test:
 
 about:
   home: https://theora.org
+  license: BSD-like
```

Comment:
@jakirkham not sure what I should add here. It's kind of BSD, but not really. Information is spread in LICENSE and COPYING.